docs/sansio.rst

```diff
@@ -253,3 +253,6 @@ Utilities
     The dict provided in *url_vars* is used in
     `URI template expansion <https://developer.github.com/v3/#hypermedia>`_.
     Appropriate URL quoting is automatically done on the values of the dict.
+
+    Enterprise GitHub users can specify their custom base URL in *base_url*.
+    By default, ``https://api.github.com/`` is used as the base URL.
```

Comment:
```suggestion
    By default, https://api.github.com/ is used as the base URL.
```
Inconsistency between the `abc` docs and here on how to handle URLs. Just chose the first style I saw. 😄

Reply:
I think, in this case, this is a variable value rather than just a URL. So I'd rather mark it as a value (inline code), not a reference for the readers to click on.